components/LinkContainer/Index.js

```diff
@@ -0,0 +1,36 @@
+import styled from "styled-components";
+import StyledLink from "@/components/Links/StyledLink";
+import SVGIcon from "@/components/Icons";
+const LinkContainer = styled.section`
+  display: grid;
+  justify-content: center;
+  grid-template-columns: 1fr 1fr;
+`;
+const StyledParagraph = styled.p`
+  color: lightgray;
+  font-size: 1.1rem;
+  margin-top: 0.4rem;
+  font-weight: bold;
+`;
+export default function LinkSection() {
+  return (
+    <LinkContainer>
+      <StyledLink href={`/events/latest`}>
```

Comment:
you don't need brackets and template literals here

```suggestion
      <StyledLink href="/events/latest">
```